src/middleware.coffee

```diff
@@ -0,0 +1,27 @@
+basic_auth = require('basic-auth')
+
+
+module.exports = 
+
+  auth: (conf) ->
+    SYSTEM_USER = conf.COUCHDB.SYSTEM_USER
+    (req, resp, next) ->
+      # look for admin credentials in basic auth, and if valid, login user as admin.
+      credentials = basic_auth(req);
+      if conf.DEV = 'true' or 
```

Comment:
that is correct. It is only allowing SYSTEM_USER if you are not in the dev environment. If you want anything else, then you should create your own auth middleware.